matter_server/server/__main__.py

```diff
@@ -98,6 +98,12 @@
     default=None,
     help="Directory where PAA root certificates are stored.",
 )
+parser.add_argument(
+    "--use-test-net-dcl",
```

Comment:
maybe the flag should be "allow-test-net-dcl" or "enable-test-net-dcl" because strictly taken its using both main and test with the toggle enabled.

Reply:
Ok, I go for `enable-test-net-dcl`.